### PR TITLE
[shared_preferences] fix documentation wording

### PR DESCRIPTION
**Pull request by Leptopoda**
The wording of the `SharedPreferences.containsKey` docs was off. This PR improves the documentation.

none. I can open an issue if really necessary.

n.a.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by Leptopoda** [comment]
Ok I don't know why `submit-queue` is failing.
Any help/insight would be great.

**Review by tarrinneal — APPROVED** [review]
nice catch! Thanks for putting this pr up!

**Comment by tarrinneal** [comment]
I always forget that one... @stuartmorgan for second approval?

File: packages/shared_preferences/shared_preferences/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2.2.2
+
+* Updates documentation for `containsKey`
```
Comment (stuartmorgan) [inline packages/shared_preferences/shared_preferences/CHANGELOG.md:3]: This is missing a period; please see the CHANGELOG style guide linked from the PR checklist.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

**Comment by stuartmorgan** [comment]
Someday we should fix all those comments; saying they are reading from/checking/etc. "persistent storage" is highly misleading given that there is at least one, and on most platforms two, layers of cache involved.

File: packages/shared_preferences/shared_preferences/CHANGELOG.md (hunk 1 above)
Comment (Leptopoda) [reply packages/shared_preferences/shared_preferences/CHANGELOG.md]: Sorry :sweat_smile:

I also just queued the changes for NEXT as I don't think a version bump is needed. Correct me if I should change it back.

Comment (Leptopoda) [reply packages/shared_preferences/shared_preferences/CHANGELOG.md]: Ok I somehow miss read 
> Non-code changes that we make to future-proof development in some way, but don't directly benefit clients. Examples of this include

And only saw the part with `Non-code changes`. 
I'll revert it again.

**Review by stuartmorgan — APPROVED** [review]

**Pull request merged**